Mention fetch() keepalive in the navigator.sendBeacon() article (#21575)
mdn/content: files/en-us/web/api/navigator/sendbeacon/index.md

legacy techniques for sending analytics, such as the use of
{{domxref("XMLHttpRequest","XMLHttpRequest")}}.

> > **Note:** For use cases that need the ability to send requests with methods other than `POST`, or to change any request properties, or that need access to the server response, instead use the [Fetch API](/en-US/docs/Web/API/Fetch_API) with [`keepalive`](/en-US/docs/Web/API/fetch#keepalive) set to true.

Reviewer: ```suggestion
> **Note:** For use cases that need the ability to send requests with methods other than `POST`, or to change any request properties, or that need access to the server response, instead use the [`fetch()`](/en-US/docs/Web/API/fetch) method with [`keepalive`](/en-US/docs/Web/API/fetch#keepalive) set to true.
```

> > **Note:** The sendBeacon() method can make POST requests only. It does not provide ability to customize the request method, provide custom request headers, or change other processing properties of the request and response. Applications that need such requests should use the fetch API with the [keepalive](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters) parameter set to true. This feature was added to fetch API to obsolete the need for sendBeacon().

Reviewer 2: ```suggestion
> **Note:** The `sendBeacon()` method can make POST requests only. It does not provide ability to customize the request method, provide custom request headers, or change other processing properties of the request and response. Applications that need such requests should use the fetch API with the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters) parameter set to true. This feature was added to fetch API to obsolete the need for `sendBeacon()`.
```

> > **Note:** The `sendBeacon()` method can make POST requests only. It does not provide ability to customize the request method, provide custom request headers, or change other processing properties of the request and response. Applications that need such requests should use the fetch API with the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/fetch#parameters) parameter set to true. This feature was added to fetch API to obsolete the need for `sendBeacon()`.

Reviewer: ```suggestion
> **Note:** `sendBeacon()` is limited to sending `POST` requests only, and does not provide any ability to add custom request headers, nor to change any properties of the request, and does not enable any access to the server response. For use cases that need the ability to send requests with methods other than `POST`, or to change any request properties, or that need access to the server response, instead use the [Fetch API](/en-US/docs/Web/API/Fetch_API) with the [`keepalive`](https://developer.mozilla.org/en-US/docs/Web/API/fetch#keepalive) parameter set to true.
```

> > **Note:** `sendBeacon()` is limited to sending `POST` requests only, and does not provide any ability to add custom request headers, nor to change any properties of the request, and does not enable any access to the server response. For use cases that need the ability to send requests with methods other than `POST`, or to change any request properties, or that need access to the server response, instead use the [Fetch API](/en-US/docs/Web/API/Fetch_API) with the [`keepalive`](/en-US/docs/Web/API/fetch#keepalive) parameter set to true.

Reviewer: ```suggestion
> **Note:** For use cases that need the ability to send requests with methods other than `POST`, or to change any request properties, or that need access to the server response, instead use the [Fetch API](/en-US/docs/Web/API/Fetch_API) with [`keepalive`](/en-US/docs/Web/API/fetch#keepalive) set to true.
```
